### PR TITLE
Auto ci ms2snoop

### PR DESCRIPTION
**Pull request by Lain-inrae**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Fixes sirius results pairing bug
Fixes pdf generation in galaxy
Releases v2.2.1

### CODE REVIEW

**Comment by Lain-inrae** [comment]
Someone @workflow4metabolomics/ms  ... ?

File: tools/ms2snoop/MS2snoop.xml [hunk 1]
```diff
@@ -48,9 +48,15 @@
...
     --fragment_match_delta '$sirius.fragment_match_delta'
     --fragment_match_delta_unit '$sirius.fragment_match_delta_unit'
-    $advenced.pdf
+    #if $advenced.pdf:
+        --pdf_path './out.zip'
+    #end if
     $advenced.debug
     $advenced.verbose
+#if $advenced.pdf:
+&& cat ./out.zip > $pdf_output
```
Comment (bgruening) [inline tools/ms2snoop/MS2snoop.xml:57]: pdf_output needs to be single quoted for security reasons: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/integration_checklist.html?highlight=quotes#creating-the-tool-wrapper-xml-file

File: tools/ms2snoop/MS2snoop.xml [hunk 2]
```diff
@@ -143,14 +149,14 @@
         </section>
     </inputs>
     <outputs>
-        <data name="frag_result_txt" format="tsv" label="${tool.name} on ${peaklist_frag_tsv.name}" />
-        <data name="pdf_output" format="zip" label="${tool.name} PDFs for ${peaklist_frag_tsv.name}">
+        <data name="frag_result_txt" format="tsv" label="${tool.name}_on_${peaklist_frag_tsv.name}.tsv" />
+        <data name="pdf_output" format="zip" label="${tool.name}_PDFs_for_${peaklist_frag_tsv.name}.zip">
```
Comment (bgruening) [inline tools/ms2snoop/MS2snoop.xml:153]: are the underscores and the extension needed? Those are not filenames, but Galaxy metadata.

**Review by bgruening — COMMENTED** [review]
@Lain-inrae cool, just one small quoting comment from my side.

File: tools/ms2snoop/MS2snoop.xml (hunk 1 above)
Comment (Lain-inrae) [reply tools/ms2snoop/MS2snoop.xml:57]: Yes, you're right, thanks!

File: tools/ms2snoop/MS2snoop.xml (hunk 2 above)
Comment (Lain-inrae) [reply tools/ms2snoop/MS2snoop.xml:153]: You're right, it's superfluous.

Comment (bgruening) [reply tools/ms2snoop/MS2snoop.xml]: Do you need the `_` in the names?

**Review by yguitton — APPROVED** [review]
Thanks for the work

**Pull request merged**